templates/map.js

```diff
@@ -393,6 +413,18 @@ var map = new maplibregl.Map({
                 },
                 attribution: `data © <a target="_top" rel="noopener" href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors.`,
             },
+            mapboxSatellite: {
+                type: 'raster',
+                tiles: [
+                    'https://api.mapbox.com/v4/mapbox.satellite/{z}/{x}/{y}.png?access_token=TODO_ADD_TOKEN'
```

Comment:
Add the project-specific token here, once we have a Mapbox account for this project.